sdk/servicebus/service-bus/test/receiveAndDeleteMode.spec.ts

```diff
@@ -516,7 +516,7 @@ describe("receive and delete", () => {
       // as your lock mode.
 
       // have to cast it - the type system doesn't allow us to call into this method otherwise.
```

Comment:
Can remove comment

Reply:
Fixed in https://github.com/Azure/azure-sdk-for-js/pull/11962/commits/11c467b6ee3d2d44d0c781d14e9b172d1a56ac0d